src/annotator/integrations/test/pdf-test.js

```diff
@@ -288,11 +289,16 @@ describe('annotator/integrations/pdf', () => {
         pdfIntegration = createPDFIntegration();
 
         document.body.appendChild(anchor.highlights[0]);
-        await triggerUpdate();
 
-        assert.equal(anchor.highlights.length, 1);
-        assert.ok(anchor.range);
-        assert.notCalled(fakeAnnotator.anchor);
+        try {
+          await triggerUpdate();
+
+          assert.equal(anchor.highlights.length, 1);
+          assert.ok(anchor.range);
+          assert.notCalled(fakeAnnotator.anchor);
+        } finally {
+          anchor.highlights[0].remove();
```

Comment:
I think this is OK for the moment, although we might want to do this as part of `PDFIntegration.destroy` as well in future.